src/Controller/CRUDController.php

```diff
@@ -41,17 +46,13 @@
 use Symfony\Component\PropertyAccess\PropertyPath;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 use Symfony\Component\Security\Csrf\CsrfToken;
+use Symfony\Contracts\Translation\TranslatorInterface;
 
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class CRUDController extends Controller
+class CRUDController extends AbstractController
```

Comment:
@jordisala1991 You forgot about interface implementation:
```suggestion
class CRUDController extends AbstractController implement ServiceSubscriberInterface
```
https://symfony.com/doc/current/service_container/service_subscribers_locators.html#defining-a-service-subscriber

Reply:
AbstractController already implements that interface